tools/osx/crosstool/cc_toolchain_config.bzl.tpl

```diff
@@ -5402,6 +5391,88 @@ def _impl(ctx):
         ],
     )
 
+    headerpad_feature = feature(
+        name = "headerpad",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = all_link_actions + [
+                    ACTION_NAMES.objc_executable,
+                    ACTION_NAMES.objcpp_executable,
+                ],
+                flag_groups = [flag_group(flags = ["-headerpad_max_install_names"])],
+                with_features = [with_feature_set(not_features = [
+                    "bitcode_embedded",
+                    "bitcode_embedded_markers",
+                ])],
+            ),
+        ],
+    )
+
+    if (ctx.attr.cpu == "ios_arm64"
+        or ctx.attr.cpu == "ios_arm64e"
+        or ctx.attr.cpu == "ios_armv7"
```

Comment:
There's also an arm7s, does that need to be covered here?

Reply:
if there's a toolchain for it (i.e. an apple_cc_toolchain) that goes with this crosstool, then possibly?

Reply:
There is not https://github.com/bazelbuild/bazel/issues/3289